documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -291,7 +294,7 @@ EOF
 <7> Specify the name of the `StorageMap` CR.
 <8> For all source providers except for {virt}, you can use either the `id` _or_ the `name` parameter to specify the source VMs. +
 {virt} source provider only: You can use only the `name` parameter, not the `id.` parameter to specify the source VMs.
-<9> Specify the VMware VM MOR, {rhv-short} VM UUID or the {osp} VM UUID.
+<9> Specify the VMware VM MOR, {rhv-short} VM UUID, the OVA UUID, or the {osp} VM UUID.
```

Comment:
it would be better in 2.6 with an example per-provider type, but until then - I think it's better to keep the order for consistency
```suggestion
<9> Specify the VMware VM MOR, {rhv-short} VM UUID, the {osp} VM UUID, or the OVA VM UUID.
```